packages/sdk/client/src/services/agent-hosting-provider.ts

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { log } from '@dxos/log';
+
+export type AgentHostingProvider = {
+  name: string;
+  BaseURL: string;
+};
+
+// TODO: Load from config or dynamically discover
+const DEFAULT_AGENT_HOSTING_PROVIDER: AgentHostingProvider = {
+  name: 'default',
+  BaseURL: 'http://localhost:8082/v1alpha1/',
```

Comment:
baseUrl
also i think we have a doc/note somewhere about port numbers in an effort to coordinate across all of our servers; i'll try to find it and link it back.